README.md

```diff
@@ -15,8 +15,8 @@ This repository is currently under development. If you want to contribute please
 You'll need [Node.js](https://nodejs.org) installed on your computer in order to build this app.
 
 ```bash
-git clone https://github.com/electron/electron-api-demos
-$ cd electron-api-demos
+git clone https://github.com/diego3g/rocketredis.git
+$ cd rocketredis
 $ npm install
```

Comment:
Since this repo is using [Yarn](https://yarnpkg.com/), we should change this to `yarn install` and the next line to `yarn dev`.

Reply:
@ruisaraiva19 noted.

Reply:
@adarshsaraogi you missed the next line, from `npm run dev` to `yarn dev` 🙂.

Reply:
@ruisaraiva19 i missed that :+1:

Reply:
Do not use yarn it is broken, when you try to use the project with it, it does not work. I did a lot of research and the problem is yarn

Reply:
@Muurilo I think you should open up a new issue with for problems with Yarn. The project is clearly using a `yarn.lock` and I've used it in several other projects.